cevelop-workspace/scope17/src/Test.cpp

```diff
@@ -63,7 +73,7 @@ void DemoFstream(){
 
 using std::filesystem::path;
 void copy_file_transact(path const & from, path const & to) {
-   path t = to.native() + ".deleteme";
+   path t = to.native() + _T(".deleteme");
```

Comment:
would a better fix to have

```
path t  = to + path(".deleteme")
```

? I need to check, why I used to.native here?

Reply:
actually

```
   path t{to};
   t += path{".deleteme"};
````
is how it should be. Can you check that?

Reply:
> actually
> 
> ```
>    path t{to};
>    t += path{".deleteme"};
> ```
> 
> is how it should be. Can you check that?

Just tested and worked. Jozef is @ Lightning Talks. He can push it once he's back.